astropy/coordinates/tests/test_representation_methods.py

```diff
@@ -159,3 +160,24 @@ def test_take(self):
         s0_take = self.s0.take((5, 2))
         assert s0_take.shape == (2,)
         assert np.all(s0_take.lon == self.s0.lon.take((5, 2)))
+
+    def test_broadcast_to(self):
+        s0_broadcast = self.s0._apply(np_broadcast_to, shape=(3, 6, 7),
+                                      subok=True)
+        assert s0_broadcast.shape == (3, 6, 7)
+        assert np.all(s0_broadcast.lon == self.s0.lon)
+        assert np.all(s0_broadcast.lat == self.s0.lat)
+        assert np.all(s0_broadcast.distance == self.s0.distance)
+        assert np.may_share_memory(s0_broadcast.lon, self.s0.lon)
```

Comment:
Yes, good idea.  Also noticed that I didn't have a default copy method in `SharedLikeNDArray`. That is now corrected as well.